core/services/health.go

```diff
@@ -50,7 +51,7 @@ func (i *InBackupHealthReport) Stop() {
 
 func (i *InBackupHealthReport) Start() {
```

Comment:
Maybe change the name `InBackupHealthReport`?

Reply:
Maybe something with `Init`, `Boot`, `StartUp`...?